src/PrestaShopBundle/Form/Admin/Sell/Product/ProductType.php

```diff
@@ -61,6 +84,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
         ;
+
+        if ($editing) {
```

Comment:
```suggestion
        if ($formIsUsedToEditAProduct) {
```
Suggestion: make it extra explicit. Other name can be used as long as it's unambiguous